docs/background/README.md

```diff
@@ -0,0 +1,14 @@
+# Kubeapps Background
+
+Big-picture explanations of higher-level Kubeapps concepts. Most useful for building understanding of a particular topic.
+
+| Background                                         | Description                                                                                      |
+| -------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| [Architecture](./architecture.md)                  | Dive into the overall Kubeapps architecture.                                                     |
+| [Kubeapps API documentation](./update-api-docs.md) | Information about Kubeapps API and how to contribute to keep Kubeapps API documentation updated. |
+
+Alternatively, our [Tutorials section](../tutorials/README.md) contain step-by-step tutorials to help outline what Kubeapps is capable of while helping you achieve specific aims, such as installing Kubeapps or using an OIDC provider.
+
+If you have a specific goal, but are already familiar with Kubeapps, take a look at our [How-to guides](../howto/README.md). These have more in-depth detail and can be applied to a broader set of features.
+
+Finally, take a look at our [Reference section](../reference/README.md) when you need to know design decisions, what functions the API supports, detailed developer guides, etc.
```

Comment:
Excellent - I like the way each section still helps you to get to the other sections when relevant.